api/cluster.go

```diff
@@ -51,6 +51,41 @@ func (s *Server) getClusterStatus(ctx *middleware.Context) {
 	response.Write(ctx, response.NewJson(200, status, ""))
 }
 
+func (s *Server) postClusterMembers(ctx *middleware.Context, req models.ClusterMembers) {
+	memberNames := make(map[string]struct{})
+	toJoin := make([]string, 0)
```

Comment:
you don't need to pre-allocate slices. just declaring them is enough. len, range, etc will work fine